### PR TITLE
footer created

### PR DESCRIPTION
**Pull request by liudmylasovetovs**
created src/css/footer.css
created src/partials/footer.html

modified src/css/base.css
modified src/css/styles.css
modified src/img/icons.svg

### CODE REVIEW

File: src/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,281 @@
+/* :root {
```
Comment (kharkov-timur) [inline src/css/footer.css:1]: Please remove this commented code

File: src/css/footer.css [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    align-items: flex-start;
+    gap: 40px;
+}
+
+footer {
+    padding: 40px 20px 20px 20px;
+    background-color: var(--black);
+}
+
+.footer-container {
```
Comment (kharkov-timur) [inline src/css/footer.css:48]: You can use another solution like this 
```
.container .footer-container {
    justify-content: space-between;
}
```

File: src/partials/footer.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<div class="footer-container">
```
Comment (kharkov-timur) [inline src/partials/footer.html:1]: Please add next classes 
<div class="container footer-container">

File: src/css/footer.css [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+    line-height: normal;
+}
+
+.footer-logo-part {
+    color: var(--green);
+    display: inline-block;
+    text-align: center;
+}
+
+.footer-adr {
```
Comment (kharkov-timur) [inline src/css/footer.css:78]: Please use transition for color

  transition-property: color;
  transition-duration: 250ms;
  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);

File: src/css/footer.css [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--black);
+    border-radius: 10px;
+    border: 1px solid rgba(255, 255, 255, 0.1);
+    width: 100%;
+    height: 100%;
+    text-decoration: none;
+    fill: var(--white);
```
Comment (kharkov-timur) [inline src/css/footer.css:165]: Please use transition for fill and background-color

File: src/css/footer.css [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+.footer-policy-link {
+    color: rgba(255, 255, 255, 0.6);
+    text-align: right;
+    font-family: var(--primary-font);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.28571;
+    letter-spacing: -0.28px;
+    text-decoration: none;
```
Comment (kharkov-timur) [inline src/css/footer.css:212]: Please use transition for color

File: src/css/footer.css [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 375px;
+    min-width: 320px;
+    margin: 0 auto;
+} */
```
Comment (mytrichq) [inline src/css/footer.css:34]: remove comments please

File: src/css/footer.css [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+
+.footer-policy-link:hover,
+.footer-policy-link:focus {
+    color: var(--green);
+}
+
+@media screen and (min-width: 768px) {
+    .container {
+        max-width: 768px;
+    }
```
Comment (mytrichq) [inline src/css/footer.css:223]: we already have such rule in base.css, remove it please

File: src/css/footer.css [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+        justify-content: space-between;
+        align-items: center;
+    }
+}
+
+@media screen and (min-width: 1280px) {
+    .container {
+        max-width: 1280px;
+        width: 90%;
+    }
```
Comment (mytrichq) [inline src/css/footer.css:263]: we already have such rule in base.css, remove it please

**Review by kharkov-timur — CHANGES_REQUESTED** [review]

**Review by kharkov-timur — APPROVED** [review]

**Pull request merged**